aten/src/ATen/native/cpu/IndexKernel.cpp

```diff
@@ -457,6 +457,74 @@ void masked_select_kernel(TensorIterator& iter, int64_t result_stride) {
     });
 }
 
+
+template <typename scalar_t>
+void cpu_hflip_vec(at::TensorIterator& iter) {
+
+  auto loop2d = [&](char** base, const int64_t *strides, int64_t size0, int64_t size1) {
+
+    static constexpr int ntensors = 3;
+    std::array<char*, ntensors> data_arr;
+    std::copy_n(base, ntensors, data_arr.data());
+    const int64_t *outer_strides = &strides[ntensors];
+
+    using Vec = Vectorized<scalar_t>;
+
+    TORCH_INTERNAL_ASSERT(sizeof(scalar_t) == -strides[0]);
+    auto stride0 = sizeof(scalar_t);
```

Comment:
I would make this constexpr and use it in place of `strides[1]` as well.
```suggestion
    constexpr auto stride = sizeof(scalar_t);
    TORCH_INTERNAL_ASSERT(stride == -strides[0] && stride == strides[1]);
```